src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -1003,5 +1004,31 @@ interface I2<out T1>
                 expected: LanguageVersion.Preview,
                 new CSharpParseOptions(LanguageVersion.CSharp8));
         }
+
+        [Fact]
+        public async Task UpgradeProjectForSealedToStringInRecords_CS8912()
+        {
+            await TestLanguageVersionUpgradedAsync(
```

Comment:
>TestLanguageVersionUpgradedAsync [](start = 18, length = 32)

Did you confirm that the test fails without chnages in CSharpUpgradeProjectCodeFixProvider?

Reply:
Yes. It failed with "No action was offered when one was expected"